xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -1435,6 +1435,7 @@ public void cdsResponseWithUpstreamTlsContext() {
     StreamObserver<DiscoveryRequest> requestObserver = requestObservers.poll();
 
     // Management server sends back CDS response with UpstreamTlsContext.
+    // Server still using envoy API V2.
```

Comment:
Not sure I understand this. Does it need a TODO comment so this can be cleaned up later?

Reply:
Currently the server only support v2, and we will support v2 servers for at least a year. I think we don't need clean up in the near future.

Reply:
I will remove this comment here.